daphne/src/lib.rs

```diff
@@ -868,6 +868,10 @@ impl<S> DapRequest<S> {
             Err(DapAbort::UnrecognizedMessage)
         }
     }
+
+    pub(crate) fn host(&self) -> String {
+        self.url.host_str().unwrap_or("").to_string()
```

Comment:
I'm pretty sure we can avoid a copy here (`.to_string()`) if return a `&str` instead of a `String`.